tests/tensorflow/nightly20220314/classifier-efficientnet.libsonnet

```diff
@@ -171,9 +171,9 @@ local tpus = import 'templates/tpus.libsonnet';
     efficientnet + v2_32 + functional + tpuVm,
     //efficientnet + v3_32 + functional,
     //efficientnet + v3_32 + convergence + timeouts.Hours(24),
-    efficientnet + v4_8 + functional + tpuVm,
-    efficientnet + v4_32 + functional + tpuVm,
-    efficientnet + v4_8 + convergence + tpuVm,
-    efficientnet + v4_32 + convergence + tpuVm,
+    //efficientnet + v4_8 + functional + tpuVm,
```

Comment:
Did you mean to remove these TPU VM configurations?

Reply:
Yes. I commented them out.

Reply:
I see -- you're removing the v4 tests.